App.js

```diff
@@ -0,0 +1,587 @@
+import React from 'react';
```

Comment:
Can this file be broken up into separate components?

Reply:
moved the styles to separate components, all other code needs to be in same components to keep it simple since its a single page app and use plugins